docs/util.md

```diff
@@ -18,13 +18,13 @@ Note, that all these classes employ TomP2P semantics. Think of these as a very t
 
 `FutureGet` extension classes: these work on all Get operations that TomP2P returns. Also includes all `BaseFuture` extensions.
```

Comment:
`FutureGet` extension classes work on all Get operations that TomP2P returns. Also includes all `BaseFuture` extensions.